service/src/test/java/bio/terra/workspace/service/spendprofile/SpendProfileBpmConnectedTest.java

```diff
@@ -27,6 +29,11 @@ public class SpendProfileBpmConnectedTest extends BaseConnectedTest {
 
   SpendProfile profile;
 
+  /** Condition used to disable these tests if run in a deployment where BPM is not configured. */
+  public boolean bpmUnavailable() {
```

Comment:
I couldn't find an easy way to cut out this duplication - using the `DisabledIf` annotation at the class level or using a function defined outside the class requires that this condition be static, which won't work here since we need to read config values.